src/certificate.ts

```diff
@@ -21,15 +22,15 @@ export async function responseCertificate(
 
   doc.pipe(res);
 
-  renderFirstPage(doc, talent, course);
+  renderFirstPage(doc, talent, course, course.lang || 'de');
```

Comment:
Instead of  `course.lang || 'de'` to define the default, I would either create a variable or set the default inside `renderFirstPage` (which you already do).

example:

```ts
const lang = course.lang || 'de' ;
```